schema/config-windows.json

```diff
@@ -32,7 +32,7 @@
                             },
                             "shares": {
                                 "id": "https://opencontainers.org/schema/bundle/windows/resources/cpu/shares",
-                                "$ref": "defs-windows.json#/definitions/cpuShares"
+                                "$ref": "defs.json#/definitions/uint16"
```

Comment:
You may want to wait on #777 before removing `cpuShares`, because the Windows folks are sounding like they'd like to return the explicit ranges to their settings ([here][1] and [here][2]).

[1]: https://github.com/opencontainers/runtime-spec/pull/777#discussion_r113536160
[2]: https://github.com/opencontainers/runtime-spec/pull/777#discussion_r113543747